tests/incremental-json-api/app/routes/application.ts

```diff
@@ -0,0 +1,36 @@
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+
+import { query } from '@ember-data/json-api/request';
+import { setBuildURLConfig } from '@ember-data/request-utils';
+import type Store from '@ember-data/store';
+import type { Document } from '@ember-data/store/-private/document';
+
+import type Author from '../models/author';
+import type Book from '../models/book';
+import type Genre from '../models/genre';
+
+setBuildURLConfig({
```

Comment:
Above this is recommended to call this function in `app/app.js`, for consistency, maybe it would be better to use the recommandation ?

Reply:
same as above, up to change still

Reply:
app.js makes the most sense